.github/workflows/bench-turborepo.yml

```diff
@@ -40,3 +47,61 @@ jobs:
             -X POST \
             -H 'Authorization: Bearer ${{ secrets.TINYBIRD_TOKEN }}' \
             'https://api.us-east.tinybird.co/v0/datasources?format=ndjson&name=turbo_benchmarks&mode=append'
+
+  time-to-first-task:
+    name: Time to First Task
+    timeout-minutes: 60
+    runs-on: ${{ matrix.os.runner }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - name: ubuntu
+            runner: ubuntu-latest
+          - name: macos
+            runner: macos-latest
+          - name: windows
+            runner: windows-latest
+
+    # Set the secret as an env var so we can use it in the script
+    env:
+      TINYBIRD_TOKEN: ${{secrets.TINYBIRD_TOKEN}}
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/setup-node
+      - uses: ./.github/actions/setup-go
+        with:
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+
+      # setup Rust on windows, because apparently setup-turborepo-environment doesn't do this
+      - name: Add rustup for windows
+        if: ${{ matrix.os.runner == 'windows-latest'}}
```

Comment:
There's a setup-rust action if that's helpful.

Reply:
Yeah.... it doesn't do the same thing as this, I didn't want to get too in the weeds of refactoring that also